backend/src/litmus_backend.py

```diff
@@ -76,26 +77,26 @@ def _pebble_layer(self) -> Layer:
 
     @property
     def _environment_vars(self) -> dict:
+        workload_version = get_litmus_version(self._container) or ""
```

Comment:
How about a unit test checking the env vars are generated correctly with the workload version?